Reference/Routing/Request-Pipeline/published-content-request-preparation.md

```diff
@@ -101,4 +101,4 @@ public void Handle(RoutingRequestNotification notification)
 
 ## Missing template?
 
-In case the router can't find a template, it will try and verify if there's route hijacking in place, if there is, it will just run the hijacked route. If route hijacking is not in place, the router will set the content to null, and run through the routing of the request again, in order for the last chance finder to find a 404.
\ No newline at end of file
+In case the router can't find a template, it will try and verify if there's route hijacking in place, if there is, it will just run the hijacked route. If route hijacking is not in place, the router will set the content to null, and run through the routing of the request again, in order for the last chance finder to find a 404.
```

Comment:
```suggestion
In case the router can't find a template, it will try and verify if there's route hijacking in place, if there is, it will run the hijacked route. If route hijacking is not in place, the router will set the content to null, and run through the routing of the request again, in order for the last chance finder to find a 404.
```